hack/verify-prerelease-lifecycle-gen.sh

```diff
@@ -2,4 +2,12 @@
 
 source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
 
-EXTRA_ARGS=--verify-only ${SCRIPT_ROOT}/hack/update-prerelease-lifecycle-gen.sh
+${SCRIPT_ROOT}/hack/update-prerelease-lifecycle-gen.sh
```

Comment:
Verify only no longer works?

Reply:
yeah, unfortunately this option was removed from code-generator/gengo